jumpscale/clients/explorer/__init__.py

```diff
@@ -19,6 +19,10 @@ def get(self, name, url, *args, **kwargs):
     def get_default(self):
         return self.get("default", url=get_config()["threebot"]["explorer_url"])
 
+    def default_addr_set(self, url):
+        threebot_cfg = get_config()["threebot"]
+        threebot_cfg["explorer_url"] = url
+        set_config("threebot", threebot_cfg)
```

Comment:
why not from jumpscale.god import j and use it here instead of explicitly import set?